kotlin-spark-api/2.4/src/main/kotlin/org/jetbrains/kotlinx/spark/api/UDFRegister.kt

```diff
@@ -0,0 +1,803 @@
+@file:Suppress("NOTHING_TO_INLINE", "DuplicatedCode", "MemberVisibilityCanBePrivate")
+
+package org.jetbrains.kotlinx.spark.api
+
+import org.apache.spark.sql.Column
+import org.apache.spark.sql.UDFRegistration
+import org.apache.spark.sql.api.java.*
+import org.apache.spark.sql.functions
+import org.apache.spark.sql.types.DataType
+import scala.collection.mutable.WrappedArray
+import kotlin.reflect.KClass
+import kotlin.reflect.full.isSubclassOf
+import kotlin.reflect.typeOf
+
+/**
+ * A shortcut for [KSparkSession.spark].udf()
+ */
+inline fun KSparkSession.udf(): UDFRegistration = spark.udf()
```

Comment:
We definitely don't need `inline`.
And why won't we put it inside `KSparkSession`?